src/elements/skeletons/TableSkeleton.tsx

```diff
@@ -38,7 +40,9 @@ const SkeletonTable: FunctionComponent<SkeletonTableProps> = ({
         justifyContent="space-around"
         gridTemplateColumns={ `repeat(${cols}, minmax(100px, 1fr))` }
       >
-        { Array(cols * rows).fill(<Skeleton height="24px" />) }
+        { Array.from({ length: cols * rows }, (_, i) => (
+          <Skeleton key={ i } height="24px" />
+        )) }
```

Comment:
@HenryQuillin I thought i came up with a cleaner solution to list rendering but forgot the key 🙄
Added it to this PR since it's fairly small and other PR is merged

Reply:
Nice! Yeah I like this better